Makefile

```
@@ -100,3 +100,9 @@ verify-bindata-v4.1.0: verify-apirequestcounts-crd
 .PHONY: verify-apirequestcounts-crd
 verify-apirequestcounts-crd:
 	diff -Naup $(APIREQUESTCOUNT_CRD_SOURCE) $(APIREQUESTCOUNT_CRD_TARGET)
+
+ALERT_MANIFESTS := bindata/assets/alerts
+.PHONY: generate
+generate:
+	jsonnet -J jsonnet/vendor -m $(ALERT_MANIFESTS) jsonnet/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
```

```suggestion
JSONNET:=go run github.com/google/go-jsonnet/cmd/jsonnet@v0.18.0
GOJSONTOYAML:=go run github.com/brancz/gojsontoyaml@v0.1.0
ALERT_MANIFESTS := bindata/assets/alerts
.PHONY: generate
generate:
	$(JSONNET) -J jsonnet/vendor -m $(ALERT_MANIFESTS) jsonnet/main.jsonnet | xargs -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml' -- {}
	find $(ALERT_MANIFESTS) -type f ! -name '*.yaml' -delete
```

What do you think if we do something like this to bring in dependent binaries?